{
  "NServiceBus.MessageId": "88fa87dc-53eb-4bf1-8e85-b16100223ce8",
  "NServiceBus.MessageIntent": "Send",
  "NServiceBus.ConversationId": "4d173e2e-28c4-4d73-b40d-b16100223ceb",
  "NServiceBus.CorrelationId": "88fa87dc-53eb-4bf1-8e85-b16100223ce8",
  "NServiceBus.ReplyToAddress": "Client",
  "NServiceBus.OriginatingMachine": "Karimi-Lab",
  "NServiceBus.OriginatingEndpoint": "Client",
  "$.diagnostics.originating.hostid": "45bca7684e756c3efdfec9bb77175207",
  "NServiceBus.ContentType": "application\/json",
  "NServiceBus.EnclosedMessageTypes": "OrderManagement.Messages.PlaceOrderCommand, OrderManagement.Messages, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
  "NServiceBus.Version": "9.0.0",
  "NServiceBus.TimeSent": "2024-04-29 02:04:39:417269 Z",
  "NServiceBus.ProcessingMachine": "Karimi-Lab",
  "NServiceBus.Retries.Timestamp": "2024-04-29 02:05:17:027593 Z",
  "NServiceBus.DeliverAt": "2024-04-29 02:04:52:302777 Z",
  "NServiceBus.ProcessingEndpoint": "Orders Management",
  "$.diagnostics.hostid": "67bc630e2a17f6ce62809f35d331118e",
  "$.diagnostics.hostdisplayname": "Karimi-Lab",
  "NServiceBus.FailedQ": "Orders Management",
  "NServiceBus.ExceptionInfo.ExceptionType": "Microsoft.Data.SqlClient.SqlException",
  "NServiceBus.ExceptionInfo.HelpLink": null,
  "NServiceBus.ExceptionInfo.Message": "Invalid object name 'dbo.Orders Management_OrderSaga'.",
  "NServiceBus.ExceptionInfo.Source": "Core Microsoft SqlClient Data Provider",
  "NServiceBus.ExceptionInfo.StackTrace": "Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid object name 'dbo.Orders Management_OrderSaga'.\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__195_0(Task`1 result)\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\n--- End of stack trace from previous location ---\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\n--- End of stack trace from previous location ---\n   at SagaPersister.GetSagaData[TSagaData](ISynchronizedStorageSession session, String commandText, RuntimeSagaInfo sagaInfo, ParameterAppender appendParameters, CancellationToken cancellationToken) in \/_\/src\/SqlPersistence\/Saga\/SagaPersister_Get.cs:line 84\n   at SagaPersister.Get[TSagaData](String propertyName, Object propertyValue, ISynchronizedStorageSession session, ContextBag context, CancellationToken cancellationToken) in \/_\/src\/SqlPersistence\/Saga\/SagaPersister_Get.cs:line 31\n   at NServiceBus.PropertySagaFinder`1.Find(IServiceProvider builder, SagaFinderDefinition finderDefinition, ISynchronizedStorageSession storageSession, ContextBag context, Object message, IReadOnlyDictionary`2 messageHeaders, CancellationToken cancellationToken) in \/_\/src\/NServiceBus.Core\/Sagas\/PropertySagaFinder.cs:line 42\n   at NServiceBus.SagaPersistenceBehavior.Invoke(IInvokeHandlerContext context, Func`2 next) in \/_\/src\/NServiceBus.Core\/Sagas\/SagaPersistenceBehavior.cs:line 77\n   at NServiceBus.LoadHandlersConnector.Invoke(IIncomingLogicalMessageContext context, Func`2 stage) in \/_\/src\/NServiceBus.Core\/Pipeline\/Incoming\/LoadHandlersConnector.cs:line 44\n   at NServiceBus.InvokeSagaNotFoundBehavior.Invoke(IIncomingLogicalMessageContext context, Func`2 next) in \/_\/src\/NServiceBus.Core\/Sagas\/InvokeSagaNotFoundBehavior.cs:line 17\n   at NServiceBus.DeserializeMessageConnector.Invoke(IIncomingPhysicalMessageContext context, Func`2 stage) in \/_\/src\/NServiceBus.Core\/Pipeline\/Incoming\/DeserializeMessageConnector.cs:line 32\n   at NServiceBus.ProcessingStatisticsBehavior.Invoke(IIncomingPhysicalMessageContext context, Func`2 next) in \/_\/src\/NServiceBus.Core\/Performance\/Statistics\/ProcessingStatisticsBehavior.cs:line 25\n   at NServiceBus.TransportReceiveToPhysicalMessageConnector.Invoke(ITransportReceiveContext context, Func`2 next) in \/_\/src\/NServiceBus.Core\/Pipeline\/Incoming\/TransportReceiveToPhysicalMessageConnector.cs:line 35\n   at NServiceBus.RetryAcknowledgementBehavior.Invoke(ITransportReceiveContext context, Func`2 next) in \/_\/src\/NServiceBus.Core\/ServicePlatform\/Retries\/RetryAcknowledgementBehavior.cs:line 25\n   at NServiceBus.MainPipelineExecutor.Invoke(MessageContext messageContext, CancellationToken cancellationToken) in \/_\/src\/NServiceBus.Core\/Pipeline\/MainPipelineExecutor.cs:line 49\n   at NServiceBus.MainPipelineExecutor.Invoke(MessageContext messageContext, CancellationToken cancellationToken) in \/_\/src\/NServiceBus.Core\/Pipeline\/MainPipelineExecutor.cs:line 68\n   at NServiceBus.LearningTransportMessagePump.ProcessFile(ILearningTransportTransaction transaction, String messageId, CancellationToken messageProcessingCancellationToken) in \/_\/src\/NServiceBus.Core\/Transports\/Learning\/LearningTransportMessagePump.cs:line 340\nClientConnectionId:f47460eb-fbcd-4b55-8498-cbfa6f2e2e60\nError Number:208,State:1,Class:16",
  "NServiceBus.TimeOfFailure": "2024-04-29 02:05:50:228969 Z",
  "NServiceBus.ExceptionInfo.Data.HelpLink.ProdName": "Microsoft SQL Server",
  "NServiceBus.ExceptionInfo.Data.HelpLink.ProdVer": "16.00.1000",
  "NServiceBus.ExceptionInfo.Data.HelpLink.EvtSrc": "MSSQLServer",
  "NServiceBus.ExceptionInfo.Data.HelpLink.EvtID": "208",
  "NServiceBus.ExceptionInfo.Data.HelpLink.BaseHelpUrl": "https:\/\/go.microsoft.com\/fwlink",
  "NServiceBus.ExceptionInfo.Data.HelpLink.LinkId": "20476",
  "NServiceBus.ExceptionInfo.Data.Message ID": "88fa87dc-53eb-4bf1-8e85-b16100223ce8",
  "NServiceBus.ExceptionInfo.Data.Transport message ID": "87b5f4e3-ce20-400e-90e1-596e0861ee3b",
  "NServiceBus.ExceptionInfo.Data.Pipeline canceled": "False"
}